pkg/apis/k0s.k0sproject.io/v1beta1/calico.go

```diff
@@ -64,6 +67,7 @@ func DefaultCalico() *Calico {
 		Overlay:                 "Always",
 		IPAutodetectionMethod:   "",
 		IPv6AutodetectionMethod: "",
+		IPTablesBackend:         "auto",
```

Comment:
Calico docs says the default is `[Default: Auto]`, note the capital `A`. dunno if it matter though but to be on safe side I'd change it to the exact same in k0s side. See https://projectcalico.docs.tigera.io/reference/felix/configuration#iptables-dataplane-configuration

Reply:
Yeah, fixed. Thanks!